solution_template/jenkins/nested/storageAccount-existing.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "outputs": {
+    "primaryEndpointBlob": {
```

Comment:
That's what I did the first time, but the deployment was failing because the ARM state machine is checking for the resource name before the resource exists, so it would fail the mainTemplate